XRTK-Core/Packages/com.xrtk.core/Services/InputSystem/MixedRealityInputSystem.cs

```diff
@@ -90,10 +91,14 @@ public MixedRealityInputSystem(MixedRealityInputSystemProfile profile) : base(pr
         private InputEventData<Vector3> positionInputEventData;
         private InputEventData<Quaternion> rotationInputEventData;
         private InputEventData<MixedRealityPose> poseInputEventData;
+        private InputEventData<IDictionary<TrackedHandJoint, MixedRealityPose>> handJointPoseInputEventData;
```

Comment:
These hand specific events don't need to be added to the input system.

Lets revisit them as gestures.